test/integration/controller-manager/cr-validation_test.go

```diff
@@ -91,7 +91,10 @@ func TestBindingPolicyValidation(t *testing.T) {
 	}{
 		{name: "junk-field-in-spec", specJSON: `{"junk": 1}`, expectOK: false},
 		{name: "junk-field-in-policy", specJSON: `{"downsync": [{"junq": true}]}`},
-		{name: "no-test-in-policy", specJSON: `{"downsync": [{"createOnly": true, "statusCollection": {"statusCollectors": ["phred"]}}]}`},
+		// Test: empty ObjectTest (no selector fields, valid fields only)
+		{name: "empty-object-test", specJSON: `{"downsync": [{"createOnly": true}]}`, expectOK: false},
```

Comment:
FYI, `, expectOK: false` is unnecessary, since `false` is the default value for that field.